tasks/steps/copy2aegisub.py

```diff
@@ -22,23 +22,41 @@ def run_step(context: dict):
         appdirs.user_data_dir("Aegisub", "", roaming=True), "automation"
     )
     pyegi_dependency_path = os.path.join(automation_path, "dependency", "Pyegi")
+    pyegi_path = os.path.join(pyegi_dependency_path, "Pyegi")
 
     # Remove previous files
     shutil.rmtree(os.path.join(automation_path, "include", "Pyegi"), ignore_errors=True)
-    shutil.rmtree(os.path.join(pyegi_dependency_path, "Pyegi"), ignore_errors=True)
+    shutil.rmtree(pyegi_path, ignore_errors=True)
     shutil.rmtree(
         os.path.join(pyegi_dependency_path, "PythonScripts"), ignore_errors=True
     )
 
     # Copy new files
-    shutil.copytree("src", automation_path, dirs_exist_ok=True)
-    shutil.copy("poetry.toml", pyegi_dependency_path)
-    shutil.copy("pyproject.toml", pyegi_dependency_path)
+    os.makedirs(os.path.join(pyegi_dependency_path, "PythonScripts"))
+    # shutil.copytree(
+    #     "src/dependency/Pyegi/PythonScripts/[sample] prefix",
+    #     os.path.join(pyegi_dependency_path, "PythonScripts", "[sample] prefix"),
+    #     dirs_exist_ok=True,
+    # )
+    shutil.copytree(
+        "src/dependency/Pyegi/Pyegi",
+        pyegi_path,
+        dirs_exist_ok=True,
+    )
+    shutil.copytree(
+        "src/include",
+        os.path.join(automation_path, "include"),
+        dirs_exist_ok=True,
+    )
+    shutil.copytree(
+        "src/autoload",
+        os.path.join(automation_path, "autoload"),
+        dirs_exist_ok=True,
+    )
```

Comment:
These should probably be modified if we decide to go with what was proposed in the previous comment.